src/microsim/schema/simulation.py

```diff
@@ -123,6 +129,43 @@ def optical_image(
             raise ValueError("truth must be a DataArray")
         # let the given modality render the as an image (convolved, etc..)
         channel = self.channels[channel_idx]  # TODO
+
+        # get the emission events for the given fluorophore
+        fluorophore_str = "EGFP"
+        em_wavelengths, em_events = get_emission_events(
```

Comment:
`Spectrum.__mul__` allows you to directly multiply two `Spectrum` objects, and it will determine the overlapping wavelength area and give you the multiplied spectra.  So it might be useful when "appyling" a filter to a fluorophore spectrum

Reply:
I looked into this. I think this could be an issue because we need both events arrays separately as well. `__mul__` will return just one spectrum.